### ISSUE TITLE
no method matching `getindex(::LightXML.XMLDocument, ::ASCIIString)`

### ISSUE THREAD

**Issue by kmsquire** [samoconnor/AWSS3.jl#1]
I'm finding I can't list buckets, because of this missing method.

```julia
julia> s3_list_buckets(aws)
ERROR: MethodError: `getindex` has no method matching getindex(::LightXML.XMLDocument, ::ASCIIString)
 in s3_list_buckets at /Users/kevin/.julia/v0.4/AWSS3/src/AWSS3.jl:192
```

Maybe AWSS3.jl is depending on an untagged (or unavailable?) version of `LightXML`?

**Comment by samoconnor** [comment]
This should be fixed in v0.0.2.
The `Associative` stuff for XML now comes from here: https://github.com/samoconnor/XMLDict.jl.

Travis tests are running now https://travis-ci.org/samoconnor/AWSS3.jl
Please let me know if you have any other issues.

**Comment by kmsquire** [comment]
Verified--thanks!
